### PR TITLE
Blog article about darktable 2.6

### PR DESCRIPTION
**Pull request by moy**
This is a rebased and cleaned-up version of #25: no intermediate commits (no need to eat disk space for obsolete screenshots), no french version.

The article is essentially finished, but contributions and proof-reading welcome.

You can contribute by suggesting stuff as comments below, or by sending pull-requests to https://github.com/moy/dtorg 's `darktable-2.6` branch. Please open an issue on https://github.com/moy/dtorg/issues before you start working on a topic, to avoid having multiple people writing the same section in parallel.

A compiled version of the article is available as a preview here:

http://matthieu-moy.fr/tmp/2018/wip-dt/2018/12/darktable-26/

A partly formatted version is visible here: https://github.com/moy/dtorg/blob/master/content/blog/2018-12-25-darktable-2.6/2018-12-25-darktable-2.6.md (image links are broken). If you want to contribute and do not know how to use Git, you can follow the link and click the edit button (pen-shaped). You need a GitHub account.

A french translation is ongoing on https://linuxfr.org/redaction/news/darktable-2-6-0 (linuxfr account needed to view and edit).

The idea is to get this PR merged at the time of the release (@TurboGit: maybe the best would be that you send a single PR with the release notes pointing to this article and the article itself).

### CODE REVIEW

**Comment by moy** [comment]
@TurboGit: how do you want to proceed for the release? My preference would be that you make a single pull-request including this article and the release notes, cross-linking to each other. Is it OK for you?

**Comment by TurboGit** [comment]
@moy: works for me. I'll prepare my pull request tomorrow. I'll see how to cross-link articles, never done that here but I suppose it is simple.

File: content/blog/2018-12-25-darktable-2.6/2018-12-25-darktable-2.6.md [hunk 1]
```diff
@@ -0,0 +1,1359 @@
+title: darktable 2.6
+author: moy
```
Comment (TurboGit) [inline content/blog/2018-12-25-darktable-2.6/2018-12-25-darktable-2.6.md]: Can you add:
`slug: darktable-26`

File: content/blog/2018-12-25-darktable-2.6/2018-12-25-darktable-2.6.md [hunk 2]
```diff
@@ -0,0 +1,1359 @@
...
+date: 2018-12-24
+lede: butterfly.jpg
+lede_author: <a href="https://matthieu-moy.fr/">Matthieu Moy</a>
+tags: announcement, darktable-release
+
+Following the tradition, darktable 2.6 was released for Christmas.
+2018 has been a year of renewal for darktable, with many major
+features introduced by recent contributors.
+
```
Comment (TurboGit) [inline content/blog/2018-12-25-darktable-2.6/2018-12-25-darktable-2.6.md]: Here, you could add:

`The announcement and release notes for this new release can be found here: [https://www.darktable.org/2018/12/darktable-260-released/](https://www.darktable.org/2018/12/darktable-260-released/).`

**Comment by moy** [comment]
Both comments applied.

**Comment by TurboGit** [comment]
@moy  : You need to close this PR as I have made one with everything included.

**Pull request closed without merging**